src/chrome/content/rules/Desjardins.xml

```diff
@@ -1,9 +1,14 @@
+
 <!--
+Disabled by https-everywhere-checker because:
+Fetch error: http://accesd.desjardins.com/ => https://accesd.desjardins.com/: (28, 'Connection timed out after 10001 milliseconds')
+Fetch error: http://accesd.affaires.desjardins.com/ => https://accesd.affaires.desjardins.com/: (28, 'Connection timed out after 10000 milliseconds')
+
```

Comment:
The http version of accessd.desjardins.com times out for me in curl and Chrome. However, when I tried to load it in Firefox, even with HTTPS Everywhere disabled, it successfully loaded and redirected to the HTTPS version.

It turns out that the site sends an HSTS header, which I had cached, so it was impossible to accurately check the HTTP version from my browser.

I think this is correct to disable because there is no HTTP version.